templates/SilverStripe/Forms/CompositeField_holder.ss

```diff
@@ -1,4 +1,4 @@
-<$Tag class="CompositeField $extraClass <% if $ColumnCount %>multicolumn<% end_if %>" id="$HolderID">
+<$Tag $AttributesHTML>
```

Comment:
`CompositeField_holder_small.ss` need to be updated as well.

`silverstripe/admin` overrides `CompositeField_holder_small.ss` and `CompositeField_holder.ss`. Those file will need to be updated in a separate PR againt admin:
* https://github.com/silverstripe/silverstripe-admin/blob/1/themes/cms-forms/templates/SilverStripe/Forms/CompositeField_holder.ss
* https://github.com/silverstripe/silverstripe-admin/blob/1/themes/cms-forms/templates/SilverStripe/Forms/CompositeField_holder_small.ss